lib/kitchen/provisioner/ansible_playbook.rb

```diff
@@ -274,7 +279,7 @@ def prepare_command
 
         # Prevent failure when ansible package installation doesn't contain /etc/ansible
         commands << [
-          sudo_env("bash -c '[ -d /etc/ansible ] || mkdir /etc/ansible'")
+          sudo_env("sh -c '[ -d /etc/ansible ] || mkdir /etc/ansible'")
```

Comment:
Please note this necessary change when working with Alpine Docker images. They come with a POSIX shell, not with Bash. Since your code is perfectly fine for POSIX shells, I changed `bash` to `sh` (which is widely aliased to `bash` on machines where Bash is available).